public/docs/_examples/gettingstarted/ts/src/tsconfig.json

```diff
@@ -1,11 +0,0 @@
-{
```

Comment:
The user needs a `tsconfig.json` - question is how do we instruct them how to create it.
This is a judgement call.

I like tsc --init because:
- most of the content of this file is default settings, you really only need three of them specified
- it is also in the copy-paste-free "from scratch" install path
- makes it more obvious that this file is the official configuration for tsc, not just some config file we invented

But it has drawbacks also:
- need to run it in the src subdir
- easy to run the wrong version of typescript which IIUC your 'npm run tsc' fixes by getting the local version